Source/Engine/ParselessLM.cpp

```diff
@@ -144,3 +144,12 @@ bool McBopomofo::ParselessLM::hasUnigrams(const std::string& key)
 
     return db_->findFirstMatchingLine(key + " ") != nullptr;
 }
+
+std::vector<std::string_view> McBopomofo::ParselessLM::getPronounciations(const std::string_view& value)
```

Comment:
Since `std::stirng_view` is basically a pointer, the returned vector doesn't have any lifetime guarantee. If the underlying string is gone, the returned vector will now contain dangling pointers. This will needs to be a `std::vector<std::string>`.

But also see my comment on the call site of this method in LanguageModelManager.mm.